### PR TITLE
Sync dev-v2.6-1.22 to dev-v2.6

### PR DESCRIPTION
**Pull request by kinarashah**

### CODE REVIEW

**Review by Oats87 — CHANGES_REQUESTED** [review]
https://github.com/rancher/kontainer-driver-metadata/pull/738

**Comment by kinarashah** [comment]
@Oats87 GH automatically syncs them, so the it pulled the commits. Could you review and approve?

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -6086,6 +6086,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
...
+			Kubernetes:                "rancher/hyperkube:v1.22.3-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.78",
+			NginxProxy:                "rancher/rke-tools:v0.1.78",
+			CertDownloader:            "rancher/rke-tools:v0.1.78",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.78",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.17.4",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.17.4",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.17.4",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.14.0",
```
Comment (aiyengar2) [inline rke/k8s_rke_system_images.go]: We should bump the flannel version here since we're planning to merge the flannel bumps to prior k8s releases as part of a direct OOB release to release-v2.6 / dev-v2.6 in https://github.com/rancher/kontainer-driver-metadata/pull/736, which means that there won't be another PR (to my knowledge) where we plan to bump Flannel in `dev-v2.6`.

```suggestion
			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
```

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: rke/k8s_rke_system_images.go (hunk 1 above)
Comment (aiyengar2) [reply rke/k8s_rke_system_images.go]: This is required for supporting Flannel in CentOS 8 nodes that may encounter an iptables segfault.

**Review by Oats87 — DISMISSED** [review]

File: rke/k8s_rke_system_images.go (hunk 1 above)
Comment (aiyengar2) [reply rke/k8s_rke_system_images.go]: This will be addressed in https://github.com/rancher/kontainer-driver-metadata/pull/739

**Review by aiyengar2 — APPROVED** [review]
